packages/data-broker/src/__tests__/DataBroker/DataBroker.test.ts

```diff
@@ -282,98 +294,38 @@ describe('DataBroker', () => {
   });
 
   describe('pull()', () => {
-    describe('input validation', () => {
```

Comment:
This is the responsibility of a different API now (`fetchAnalytics()` etc), test cases have moved there